java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1187,6 +1276,52 @@ public final Builder append(double value) {
       return this;
     }
 
+    public final Builder append(BigDecimal value) {
+      return append(value, null);
+    }
+
+    //
```

Comment:
This public method needs Javadoc explaining the details with scale handling/expectations, etc.

Reply:
It's done :)